test/info.py

```diff
@@ -56,6 +56,7 @@
     ("k20dx_mimxrt1020_evk_qspi_if",                True,       0x8000,     "bin"       ),
     ("k20dx_mimxrt1050_evk_hyper_if",               True,       0x8000,     "bin"       ),
     ("k20dx_mimxrt1050_evk_qspi_if",                True,       0x8000,     "bin"       ),
+    ('k20dx_ep_agora_if',                           False,      0x0000,     "bin"       ),
```

Comment:
Do we need to match this to the other `k20dx` entries?

For example:
```
('k20dx_ep_agora_if',                           True,      0x8000,     "bin"       ),
```

Reply:
I would say yes